docs/GettingStarted.md

```diff
@@ -412,6 +412,23 @@ for the first time:
 Remember that the debug mode may affect your application performance and so it must not be used
 in a production environment.
 
+### Environment and tags
+
+By default, the trace agent (not this library, but the program running in
+the background collecting data from various clients) uses the tags
+set in the agent config file, see our
+[environments tutorial](https://app.datadoghq.com/apm/docs/tutorials/environments) for details.
+
+These values can be overridden at the tracer level:
+
+    Datadog.tracer.set_tags('env' => 'prod', 'component' => 'core')
```

Comment:
should we put the `'component' => 'core'` in this part of the doc? just to be sure it doesn't create misunderstandings, since only the `env` tag is required.